### ISSUE TITLE
[Bug] Controller's graph handler returns invalid JSON data

### ISSUE THREAD

**Issue by moisseev** [vstakhov/rspamd#615]
~~~
[[{"x":1430647200,"y":NaN},{"x":1430650800,"y":NaN},{"x":1430654400,"y":0.0},{"x":1430658000,"y":NaN},
~~~
`NaN` is not a valid JSON token and should be replaced with `null`.

**Comment by vstakhov** [comment]
This should be fixed now. By the way, I would appreciate any reports about how rrd feature works in rspamd. I have not absolutely tested it yet, I'm afraid.

**Comment by moisseev** [comment]
Fixed, thanks! 

My report is here: https://github.com/vstakhov/rspamd/issues/405#issuecomment-188132960

**Comment by moisseev** [comment]
I'm getting strange JSON data from controller using URI
like ```http://hostname:11334/graph?password=XXX&type=hourly```
except `type=monthly`.

Graphs created with D3 using JSON:
![json-hourly](https://cloud.githubusercontent.com/assets/2275981/15403879/a85a22fe-1e03-11e6-8092-5bac1a47d313.png)
![json-daily](https://cloud.githubusercontent.com/assets/2275981/15403890/b0150dd8-1e03-11e6-9668-1826f2359a0e.png)
![json-weekly](https://cloud.githubusercontent.com/assets/2275981/15403894/b4f9352c-1e03-11e6-9236-1421d4b5af44.png)
![json-monthly](https://cloud.githubusercontent.com/assets/2275981/15403896/b83e850c-1e03-11e6-8fef-16f0d7913181.png)
Graphs created with `rrdcgi` directly from RRD (pink vertical lines mean data is unavilable):
![rrd](https://cloud.githubusercontent.com/assets/2275981/15403913/c7373676-1e03-11e6-9076-7554f0619adb.png)

[JSON.zip](https://github.com/vstakhov/rspamd/files/273141/JSON.zip)
[rspamd.rrd.zip](https://github.com/vstakhov/rspamd/files/273147/rspamd.rrd.zip)

**Comment by moisseev** [comment]
Fixed with be8b548ac0dbb9fb0ee65dbffc2f0ebe14f6cc59
